src/smc-webapp/app/page.tsx

```diff
@@ -88,6 +88,11 @@ const PAGE_STYLE: React.CSSProperties = {
 
 const positionHackHeight = `${NAV_HEIGHT - 4}px`;
 
+// smc-util/theme is set in ./render.coffee
+const BASE_URL = global["BACKEND"]
+  ? require("smc-util/theme").DOMAIN_NAME
+  : require("../misc").BASE_URL;
```

Comment:
well, the hardcoded domain name is a fallback. please don't use it. the backend should use the site settings from the database, which will set this domain name as a fallback – see `get_server_settings_cached` method. I wonder why you set a domain name at all, though. 

and more general, is this really being used by the backend? maybe just don't render this tab, no need to do so.

Maybe it's best to just import `APP_BASE_URL` from misc and that's it. … and please import with `import` at the top, not `require` inline.

Reply:
I copied the code used for the info page. I just pushed the change you asked for.